tests/benchmarks/mini-js/Pulumi.yaml

```diff
@@ -0,0 +1,6 @@
+name: mini-js
+runtime:
+  name: nodejs
+  options:
+    typescript: false
```

Comment:
It doesn't, no! This was a surprising find for me. We could consider that autodetect, WDYT?

I was debating this in my head but opted for just changing templates for now https://github.com/pulumi/templates/pull/312

This means old projects will not get a speed up, only new projects.

Reply:
I'm not sure if we'd want to do that autodetection as handling edge cases could be messy:

* In a monorepo, node_modules could contain plain TS (`yarn link`/lerna/etc.) files, a way of skipping transpile steps
* Even in a "plain" project folder, the root file could be index.js but it could import a .ts file.

I think to reliably detect "ts", we'd have to implement Node module resolution & import. Easier to allow opt out, I think?

Reply:
Did not know that JS could import typescript. Yes let's not pursue auto-detection if we do not know a reliable way to do it. Thanks!